air/src/proof/ood_frame.rs

```diff
@@ -9,18 +9,25 @@ use utils::{
     SliceReader,
 };
 
-// TYPE ALIASES
-// ================================================================================================
-
-type ParsedOodFrame<E> = (Vec<E>, Vec<E>);
+use crate::LagrangeKernelEvaluationFrame;
 
 // OUT-OF-DOMAIN FRAME
 // ================================================================================================
+
+/// Represents an [`OodFrame`] where the trace and constraint evaluations have been parsed out.
+pub struct ParsedOodFrame<E> {
+    pub ood_trace_evaluations: Vec<E>,
+    pub ood_lagrange_kernel_trace_evaluations: Option<Vec<E>>,
+    pub ood_constraint_evaluations: Vec<E>,
```

Comment:
minor nit: do we need the `ood` prefix here?

Reply:
Removed